apps/web/src/components/conditions/Conditions.tsx

```diff
@@ -3,30 +3,34 @@ import styled from '@emotion/styled';
 import { useMemo } from 'react';
 import { Control, Controller, useFieldArray, useForm, useWatch } from 'react-hook-form';
 
-import { FILTER_TO_LABEL, FilterPartTypeEnum } from '@novu/shared';
+import { FilterPartTypeEnum } from '@novu/shared';
 
 import { Button, colors, Dropdown, Input, Select, Sidebar, Text, Title, Tooltip } from '../../design-system';
 import { ConditionPlus, DotsHorizontal, Duplicate, Trash, Condition, ErrorIcon } from '../../design-system/icons';
 import { When } from '../utils/When';
-import { ConditionsContextEnum, ConditionsContextFields, IConditions } from './types';
+import { DataSelect, IConditions, IConditionsForm, IConditionsProps, IFilterTypeList } from './types';
+import { OnlineConditionRow } from './OnlineConditionRow';
+import { DefaultGroupOperatorData, DefaultOperatorData } from './constants';
+import { PreviousStepsConditionRow } from './PreviousStepsConditionRow';
 
-interface IConditionsForm {
-  conditions: IConditions[];
-}
 export function Conditions({
```

Comment:
on smaller screens the AND/OR field is almost not visible
![Screenshot 2023-09-20 at 18 43 37](https://github.com/novuhq/novu/assets/2607232/de19db6d-aafc-4dd6-82c2-db7452819ea0)